servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -192,7 +202,7 @@ private UiFileInfo getUiFileInfo(TachyonFS tachyonClient, TachyonURI filePath)
    * @throws IOException
    */
   private UiFileInfo getUiFileInfo(TachyonFS tachyonClient, int fileId, TachyonURI filePath)
-      throws FileDoesNotExistException, IOException {
+      throws FileDoesNotExistException, NotFoundException, IOException {
```

Comment:
it is thrown by codes in `getUiFileInfo` itself. Yes, it can not replaced with `NotFoundException`.